monai/fl/client/monai_algo.py

```diff
@@ -135,18 +182,25 @@ def get_weights(self, extra=None):
         self.phase = FlPhase.GET_WEIGHTS
         if self.trainer:
             weights = self.trainer.network.state_dict()
```

Comment:
I would suggest to use the `get_state_dict` function of MONAI utils to enhance the logic here:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/utils.py#L382

Thanks.

Reply:
Thanks. changed it.